piksi_tools/console/settings_view.py

```diff
@@ -212,6 +227,40 @@ def _factory_default_button_fired(self):
                         + "Are you sure you want to reset to factory defaults?"
     confirm_prompt.run(block=False)
 
+  def _settings_save_to_file_fired(self):
```

Comment:
I guess they can always "reset to defaults" or worst case do a full bootload. @denniszollo @cbeighley , what do you guys think?